context/app/static/js/components/entity-search/sidebar/DatasetConfigureSearchTable/hooks.js

```diff
@@ -24,13 +24,17 @@ function selectedItemsReducer(state, { type, payload }) {
   }
 }
 
-function useGroupedMetadataFieldConfigs() {
+function useGroupedFieldConfigs() {
+  const { initialFields, initialFacets } = useStore();
   const metadataFieldConfigs = useMetadataFieldConfigs();
-  return Object.entries(metadataFieldConfigs).reduce((acc, [metadataFieldName, metadataFieldConfig]) => {
+
+  // Order matters. Field configs defined in initialFields/initialFacets will be overwritten
```

Comment:
```suggestion
  // Order matters. Field configs defined in initialFields/initialFacets will be overwritten.
```
and periods on sentences.